### DISCUSSION TITLE
energy of decomposition about protein-ligand system

### DISCUSSION THREAD

**Discussion by sf-polaris** [Valdes-Tresanco-MS/gmx_MMPBSA#434] [Q&A] [answered]
Hi! The ligand can interact with protein with different interactions, such as π-π interaction，cation-π interaction, hydrogen bond and so on.  And decomposition divides into  internal item、van del
 waals item、electrostatic item 、polar and nonpolar . I want to know how the different interactions match different energy of decomposition. For example, what is the π-π interaction divided ?

**Comment by marioernestovaldes** [comment, marked as answer]
I would say if you want to have a clearer idea on how these different interactions contribute to the binding, you would need to do some mutagenesis studies removing one interaction at a time... In the decomposition scheme, there is no way to rigorously calculate the energies corresponding to each of these interactions although,  and this is the interesting part, they are well described by the force field terms ( http://archive.ambermd.org/202105/0094.html)

**Reply by sf-polaris** [reply]
thank you!
